.editorconfig

```
@@ -0,0 +1,5 @@
+root = true
+
+[*.{c,h,py}]
+indent_style = space
+indent_size = 4
```

```suggestion
indent_size = 4

[*.yml]
indent_style = space
indent_size = 4
```

I had to add this in my local .editorconfig to make my editor behave correctly, I figure it might be worth to have?

---

Thanks @schopin-pro ! I added your suggestion. I changed the `indent_size` to `2` though since that seems to be the convention currently.

---

```
  yes: please
```